### PR TITLE
Integration tests for Most Watched Page

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #NUMBER

**Overall change:**
Add integration tests for most watched page

**Code changes:**
- Add integration tests
- Update snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/integration/pages/mostWatchedPage/ampTests.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { runCoreAmpTests, runAmpAnalyticsTests } from '../../common';
+import runCrossPlatformTests from './crossPlatformTests';
+
+export default service => {
```
Comment (jroebu14) [inline src/integration/pages/mostWatchedPage/ampTests.js]: `service` is a global in all int test files

File: src/integration/pages/mostWatchedPage/crossPlatformTests.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { runCommonCrossPlatformTests } from '../../common';
+
+export default service => {
+  runCommonCrossPlatformTests(service);
+
+  it('I can see the headline', () => {
```
Comment (jroebu14) [inline src/integration/pages/mostWatchedPage/crossPlatformTests.js]: there's a function for this in `src/integration/common/mainHeading.js`

File: src/integration/pages/mostWatchedPage/ampTests.js (hunk 1 above)
Comment (RichardPK) [reply src/integration/pages/mostWatchedPage/ampTests.js]: Aye it seems like we have been passing it down in other integration tests, but there's no reason to.

@pharingee - without the passing of service as props works fine https://github.com/bbc/simorgh/commit/15ce277aba750892470fc0c2a150cebe46a9ca13

**Review by RichardPK — APPROVED** [review]
Comment on passing of service being unnecessary (as in this commit I tried https://github.com/bbc/simorgh/commit/15ce277aba750892470fc0c2a150cebe46a9ca13) - but, as we do that in other integration tests, up to you. 🤷‍♂️ 

Otherwise LGTM

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**